base.css

```diff
@@ -1,10 +1,89 @@
-@import "intro.css";
-@import "index.css";
-
 * {
   box-sizing: border-box;
   margin: 0;
   padding: 0;
   font-size: 100%;
   font-family: "Roboto", sans-serif;
 }
+
+.gridLayout {
+  position: absolute;
```

Comment:
Why `position: absolute` here? i've tried removing it and nothing really changes. 
Be careful with aboslute positioning. When you think about a layout, most of the times you can achieve it with grid and flex.

Reply:
I needed to experiment for a while since i had an element with an y scroll overlay in out of the flow to add the video background. I had experimented a bit and simply forgot to remove the position atribute here. ;)

Position absolute corrected    (removed)